Editor/UI/ProjectAuditorWindow.cs

```diff
@@ -352,6 +353,18 @@ static void OnLoad()
                     },
                     analyticsEvent = (int)ProjectAuditorAnalytics.UIButton.ShaderCompilerMessages
                 });
+
+                ViewDescriptor.Register(new ViewDescriptor
+                {
+                    category = IssueCategory.Texture,
+                    name = "Texture",
+                    menuOrder = 6,
+                    showInfoPanel = true,
+                    // type = typeof(SummaryView),
+                    // analyticsEvent = (int)ProjectAuditorAnalytics.UIButton.Summary
```

Comment:
let's add a new analyticsEvent for this.

Reply:
Added in the new Texture analytics event